templates/Dockerfile.template

```diff
@@ -42,32 +42,22 @@ RUN set -ex; \
 	rm -r "$GNUPGHOME"; \
 	apt-key list
 
-RUN echo 'deb http://www.apache.org/dist/cassandra/debian {{ $cassandra.Version }}x main' >> /etc/apt/sources.list.d/cassandra.list
-
-RUN set -x \
-	&& apt-get update && apt-get install -y --no-install-recommends curl && rm -rf /var/lib/apt/lists/* \
-	&& curl -s http://dl.bintray.com/apache/cassandra/dists/{{ $cassandra.Version }}x/main/binary-amd64/Packages | awk '/Version/ { print "export CASSANDRA_FULLVERSION=" $2 }' > /tmp/cassandra-version \
-	&& apt-get purge -y --auto-remove curl
-
+RUN echo {{ $cassandra.Version }} | sed 's-\([^.]\+\)\.\([^.]\+\).*$-deb http://www.apache.org/dist/cassandra/debian \1\2x main-' >> /etc/apt/sources.list.d/cassandra.list
 
 ENV CASSANDRA_VERSION {{ $cassandra.Version }}
 
 RUN set -x; \
-	. /tmp/cassandra-version \
-	&& if [ "${CASSANDRA_FULLVERSION%%.*}" = "3" ]; then \
+	if [ "${CASSANDRA_VERSION%%.*}" = "3" ]; then \
```

Comment:
You can also do something similar like mysql has:
https://github.com/GoogleCloudPlatform/mysql-docker/blob/master/templates/Dockerfile.template#L30